<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindDialog</class>
 <widget class="QDialog" name="FindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find row</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="nameRadioButton">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="OGRNRadioButton">
       <property name="text">
        <string>OGRN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="addressRadioButton">
       <property name="text">
        <string>Address</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="nameOwnerRadioButton">
       <property name="text">
        <string>Owner name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="phoneRadioButton">
       <property name="text">
        <string>Phone</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="countProductRadioButton">
       <property name="text">
        <string>Product count</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="countPostRadioButton">
       <property name="text">
        <string>Post count</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="priceRadioButton">
       <property name="text">
        <string>Price</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="tableLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Enter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="enterLineEdit"/>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FindDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
